apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -4,6 +4,7 @@
 {{#inline}}
 <span>
 {{/inline}}
+    <span aria-label="multiple choice options">
```

Comment:
Another place where `<fieldset>` may be appropriate.

Reply:
Updated to `<fieldset>` and `<legend>` in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).